README.md

```diff
@@ -159,6 +159,16 @@ not present in the project's `gradle.properties`:
 
 -------
 
+#### Applying to an application module
+
+Application module merges resources into a binary format which is not supported yet.
+
+See [tracking issue](https://github.com/cashapp/paparazzi/issues/107) for more details.
+
+To work around this move the tested resources (or most of the app) into a library module and use Paparazzi on that.
+
+-------
```

Comment:
I've had a change of heart.  Let's move issues to the tracker and out of the README.  #107 details the issue (and a subsequent PR with that piece will show the error at runtime).